src/tools/dart-create.md

```diff
@@ -56,6 +56,14 @@ For further information on command-line options, use the `--help` flag:
 $ dart create --help
 ```
 
+{{site.alert.version-note}}
```

Comment:
Thanks for adding this, I forgot about it! If this PR will take a while, perhaps this can be pulled out.